sdk/communication/azure-communication-network-traversal/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1package-2021-06-21-preview
```

Comment:
This should be `1.0.0b1 (2021-07-09)` or what ever the date this will be merged/released.